virttest/cpu.py

```diff
@@ -1607,6 +1609,68 @@ def get_cpu_info_from_virsh(params):
         virsh.undefine(name, ignore_status=True, timeout=timeout)
 
 
+def get_amd_cbit_position():
```

Comment:
Shall we introduce a separated module, e.g. `coco` (acronym of confidential computing) for placing the relevant functionalities? therefore, we could group them by hardware platforms, like `coco.sev`, `coco.tdx`, etc., and differentiate the TEE operations from the CPU specific ones.